src/packing_queue/PackingQueue.jsx

```diff
@@ -203,6 +242,32 @@ const PackingQueue = () => {
     await fetchSearch(getSortFromModel(sortPackHistoryModel), 0, "", "");
   }
 
+  const onPendingClearClick = async () => {
```

Comment:
Might be a question for @jarrilla but do we want to clear the search as we navigate tabs?

Reply:
I made it so that the searches between history and pending were separate, but it does persist across sessions.